WebContent/js/utilities/fileHelper.js

```diff
@@ -0,0 +1,21 @@
+
+export function parseFileName(file) {
+    if (file === '') {
+        return { DSName: '', DSMember: '' };
+    }
+
+    let end = file.indexOf('(');
+    if (end === -1) {
+        end = file.length;
+    }
+    return { DSName: file.substring(0, end),
+        DSMember: file.substring(file.indexOf('(') + 1, file.indexOf(')')) };
+}
+
+export function hasMember(file) {
+    return file !== undefined && file.indexOf('(') > 1;
+}
+
+export function updateDSName(file, newDSName) {
```

Comment:
As far as I can tell this function is only used by the editor reducer therefore it should stay in that file for simplicity sake. As it's only string manipulation and used in one place it probably doesn't even need to be a function.

Reply:
i guess for now we can do that, moved in editor.js reducer